web-ng/src/app/components/layer-dialog/layer-dialog.component.ts

```diff
@@ -49,6 +49,7 @@ export class LayerDialogComponent implements OnDestroy {
   lang: string;
   layer?: Layer;
   layerName!: string;
+  layersSize!: number;
```

Comment:
Does this refer to the number of layers? If so, `layerCount` would be more idiomatic.

Reply:
This comment is still valid. Thanks!